java-api-core/src/com/kinvey/java/core/MediaHttpUploader.kt

```diff
@@ -928,7 +928,7 @@ constructor (mediaContent: AbstractInputStreamContent,
     /**
      * Returns the HTTP content of the media to be uploaded.
      */
-    fun getMediaContent(): HttpContent = mediaContent
+    //fun getMediaContent(): HttpContent = mediaContent
```

Comment:
Why have `getMetadata()` and `getMediaContent()` been commented out? Would this be a breaking change?

Reply:
Fixed.